src/components/reference/_ci-scheduling.mdx

```diff
@@ -1,11 +1,11 @@
 The following table is a summary of methods and resources to set up schedules for different CI providers.
 
-| CI provider | Where to set schedule | Resource |
-| ----------- | --------------------  | -------  |
-| GitHub Actions | Within `semgrep.yml` file | [Sample code snippet](/semgrep-ci/sample-ci-configs#sample-github-actions-configuration-file) |
-| GitLab CI/CD | Within GitLab CI/CD interface |[Official documentation](https://docs.gitlab.com/ee/ci/pipelines/schedules.html) |
-| Jenkins | Within Jenkins interface |  [Official documentation](https://www.jenkins.io/doc/book/pipeline/running-pipelines/#scheduling-jobs-in-jenkins) |
-| BitBucket Pipelines | Within BitBucket Pipelines interface | [Official documentation](https://support.atlassian.com/bitbucket-cloud/docs/pipeline-triggers/) |
-| CircleCI | Within CircleCI interface | [Official documentation](https://circleci.com/docs/scheduled-pipelines#get-started-with-scheduled-pipelines-in-circleci) |
-| Buildkite | Within Buildkite interface | [Official documentation](https://buildkite.com/docs/pipelines/scheduled-builds) |
-| Azure Pipelines | Within Pipelines interface (recommended) | [Official documentation](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml) |
+| CI provider | Where to set schedule |
+| ----------- | --------------------  |
+| GitHub Actions | [`semgrep.yml` file](/semgrep-ci/sample-ci-configs#sample-github-actions-configuration-file) |
```

Comment:
I can see a user clicking on the link of the yaml file and thinking they have to edit that rather than their own yaml file, or clicking on [Jenkins interface] and wondering why the link is to documentation rather than a the menu in Jenkins, for example.

Reply:
Yeah, I don't think this is great right now; my aspirational goal is to set these up myself at a later point and then draft sample code snippets for the end-user...but again, at a later date

Reply:
The code snippets aren't possible for several of those CI providers; some of them prefer that the user set the schedule through a menu in their interface. There's no `cron: * * * *` value in their CI yaml files that you can snippet. If you want to compress the information, I suggest that the table header be something like this:

| CI provider | How to set schedule |
| ----- | ----  | 
| GitHub Actions | In your semgrep.yml file, refer to Sample CI Configs ... |
| Jenkins | Refer to Jenkins documentation (entire cell is a link) |
| CircleCI | Refer to CircleCI documentation (entire cell is a link) |

and so on.